src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -182,32 +182,35 @@
     </Copy>
   </Target>
 
-  <!-- Workaround for VS execution:  This will form the same list and copy the same files as
-       copied via RunTests script so VS can work when the test dir is initially clean.
+  <!-- This forms the same list and copy the same files as copied via RunTests script, when running locally.
```

Comment:
I'm not convinced this is the same list that is in RunTests scripts, as they use different item groups, we should ensure they are the same by using the same item group and only modify it in one common target.